render/template_helpers.go

```diff
@@ -0,0 +1,60 @@
+package render
+
+import (
+	"html/template"
+
+	"github.com/gobuffalo/tags"
+)
+
+var assetMap map[string]string
```

Comment:
I feel like we're opening ourselves up to a race condition. we're not synchronizing access to this map anywhere.